spec/Cabin/InteriorLights.vspec

```diff
@@ -56,6 +56,11 @@
 
 - Spotlight.IsPassengerOn:
```

Comment:
Spotlight is some interior light, and haveing only one implies, it is only on one seat (and driver)?
Does it make sense "generally"? There might be lights over rear seats also... But i think this comment is general, not specifically tied to this PR...

Reply:
Remove Spotlight from the PR and create ticket.

Reply:
@SebastianSchildt: looking again at the PR, it is defined for each row. Maybe the solution is not so bad at the end. For every row you have shared, driver side, and passenger side. Option would be to rename it to left/right, so that it matches how we model it in other places.

Reply:
> Spotlight is some interior light, and haveing only one implies, it is only on one seat (and driver)?

Yes, if only one is modelled it might not be enough.  But I want to highlight as I have done before that I think in general we should model a data hierarchy, and not primarily the design of the physical car.  Here, to my understanding there is only one _branch_ named Spotlight, that is true,  but Spotlight is here a category more than a physical thing?   The branch could have many leaves under it, thus implying many lights that are of type Spotlight, do you agree with my thinking?